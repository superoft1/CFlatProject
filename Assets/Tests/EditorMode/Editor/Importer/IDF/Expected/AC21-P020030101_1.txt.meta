fileFormatVersion: 2
guid: b97e87f30e687f74283c00600a7b5304
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
