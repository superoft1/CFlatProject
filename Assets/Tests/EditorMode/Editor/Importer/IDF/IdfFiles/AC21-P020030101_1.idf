  3275     1     1     2  1000     0     1    12     0    30     24    14    18     1
     0     0     1     1     1   100     0     0   200     0      0     0     2    24
    70     1     0   100     0   100 061086    2     0    80      1     0     0     7
     1     1     1     1     1     1     1     1     1     0      0     4     0     0
     0     0     1     1     2     1    55     2     0     0      0     0     0     0
  8000     0     0     0     0     0     1     0     0  1100      0     0     0     0
     0     0     0     0     0     0     0     0     0     0      0     0     0     0
     3     0  3030     0     0     0     0     0     0     0  111111    0     1    15
     0     1     0   200     0     0     0   100     0     0      5     0     0     0
     0     0     0     0     0     0     0     0     0     0      0     0     0     0
-102  /ac21p0200301             
  -1  01.i*                     
-103  /\\LTCSPV\NSR             
  -1  P\n2\iso\ref_             
  -1  pip\symbol.li             
  -1  b                         
-105  /pdsidf.err               
-115  /ac21p0200301             
  -1  01.sht                    
-122  /\\LTCSPV\NSR             
  -1  P\n2\iso\ref_             
  -1  pip\user.mld              
-123  /\\LTCSPV\NSR             
  -1  P\n2\iso\ref_             
  -1  pip\dsketch.d             
  -1  df                        
-124  /\\LTCSPV\NSR             
  -1  P\n2\iso\ISO_             
  -1  DETAIL\7.AC21             
  -1  \AC21-NSRP2IS             
  -1  O_V01.CEL                 
   3      -6      25       5       1       0       0       0       0       0       0       0       0       0
   3     -25       0       0       0       0       0       0       0       0       0       0       0       0
   3    -702       0       0       0       0       0       0       0       0       0       0       0       0
   3    -703       0       0       0       0       0       0       0       0       0       0       0       0
-201  E                         
-202  N                         
-203  W                         
-204  S                         
-205  EL +                      
-206  EL -                      
-207  @                         
-208  CONN TO                   
-209  CONN ON                   
-210  F                         
-211  G                         
-212  B                         
-213  STEM                      
-221                            
-222  SLOPE                     
-243  ECC FLAT                  
-244  UP                        
-245  DOWN                      
-250                            
-253                            
-255  SHT                       
-270                            
-271                            
-272                            
-290  INSUL:                    
-291  TRACE:                    
-292  PAINT:                    
-293                            
-294                            
-295                            
-296                            
-297                            
-298                            
-300  SHOP MATERIAL             
  -1  S                         
-302  NO.                       
-303  PART & DESCRI             
  -1  PTION                     
-304  SIZE                      
-305  SYMBOL                    
-308  FITTINGS                  
-310  FIELD MATERIA             
  -1  LS                        
-313  VALVES                    
-317                            
-318                            
-321  NO.                       
-326  1HF-BEND                  
-328                            
-330  IN.                       
-359                            
-400                            
-401                            
-402                            
-403                            
-404                            
-405                            
-406                            
-407                            
-410                            
-411                            
-451                            
-498                            
-538  1                         
  -6  P-020-0301-01             
  -1  -                         
 -11  B1AQ02                    
 -14  04/06/15                  
 -15  HC 65                     
 -17  NONE                      
 -30  M/L DWG NO.:              
  -1  AC21-P-020-03             
  -1  01-02                     
 300         11943        9261         103           0           0           0    0       0,   0,       0,    ,       0       0
  45         26201        4731       80318       -4277        4731       80318  256       1,   0, 1110000,TEBW,       0       1
   0             0           0           0           0           0           0  256       0,   0,       0,    ,       0       0
  46         -4277        4731       80318       -4277        4731      110797  256       1,   0, 1110000,TEBW,       0       1
 120         -4277        4731      110797       -4277        4731      110797  256       0,   0,       0,WW  ,       0       1
 100         -4277        4731      110797       -4277        4731      219360  256       1,   0,   10000,    ,       0    1011
 -39         5    9             
  -1  308      48               
  -1   11231      4             
  -1  8                         
 120         -4277        4731      219360       -4277        4731      219360  256       0,   0,       0,WW  ,       0       1
  35         -4277        4731      219360       -4277        4731      280318  256       2,   0,   10000,ELBW,    9000    1011
 -39         3   16             
  -1  882      48               
  -1   11231      4             
  -1  8                         
  36         -4277        4731      280318      -65236        4731      280318  256       0,   0,   10000,ELBW,    9000    1011
 120        -65236        4731      280318      -65236        4731      280318  256       0,   0,       0,WW  ,       0       1
 100        -65236        4731      280318     -206000        4731      280318  256       1,   0,   10000,    ,       0    1011
 -39         5    3             
  -1  054      48               
  -1   11231      4             
  -1  8                         
 150       -206000        4731      280318           0           0           0  256       3,   0,   10000,01HG,       0     202
 -39         8    2             
  -1  192      48               
  -1   11231      4             
  -1  8                         
 150       -206000        4731      280318           0           0           0  256       4,   0,   10000,    ,       0     602
 -22  DOWN                      
 -39         8    2             
  -1  193      48               
  -1   11231      4             
  -1  8                         
 -71  SUPPORT ON CI             
  -1  VIL STANCHION             
 149       -486748        4731      280318           0           0           0  256       0,   0,       0,FLOW,       0      10
 100       -206000        4731      280318     -767497        4731      280318  256       1,   0,   10000,    ,       0    1011
 -39         5    3             
  -1  054      48               
  -1   11231      4             
  -1  8                         
 120       -767497        4731      280318     -767497        4731      280318  256       0,   0,       0,WW  ,       0       1
  35       -767497        4731      280318     -828456        4731      280318  256       2,   0,   10000,ELBW,    9000    1011
 -39         3    6             
  -1  367      48               
  -1   11231      4             
  -1  8                         
 150       -828456        4731      280318           0           0           0  256       3,   0,   10000,01HG,       0     202
 -39         8    2             
  -1  226      48               
  -1   11231      4             
  -1  8                         
 150       -828456        4731      280318           0           0           0  256       5,   0,   10000,    ,       0     602
 -22  DOWN                      
 -39         8    2             
  -1  225      48               
  -1   11231      4             
  -1  8                         
 -71  SUPPORT ON CI             
  -1  VIL STANCHION             
 150       -828456        4731      280318           0           0           0  256       6,   0,   10000,01HG,       0     402
 -78  MS008V                    
 -22  WEST                      
 -39         8    2             
  -1  224      48               
  -1   11231      4             
  -1  8                         
  36       -828456        4731      280318     -828456       47835      323422  256       0,   0,   10000,ELBW,    9000    1011
 120       -828456       47835      323422     -828456       47835      323422  256       0,   0,       0,WW  ,       0       1
 100       -828456       47835      323422     -828456       86768      362358  256       1,   0,   10000,    ,       0    1011
 -39         5    9             
  -1  309      48               
  -1   11231      4             
  -1  8                         
 120       -828456       86770      362360     -828456       86770      362360  256       0,   0,       0,WW  ,       0       1
  35       -828456       86770      362360     -828456      104731      380318  256       7,   0,   10000,ELBW,    4500    1011
 -39         3   16             
  -1  883      48               
  -1   11231      4             
  -1  8                         
  36       -828456      104731      380318     -828456      130130      380318  256       0,   0,   10000,ELBW,    4500    1011
 120       -828456      130130      380318     -828456      130130      380318  256       0,   0,       0,WW  ,       0       1
 100       -828456      130130      380318     -828456      629030      380318  256       1,   0,   10000,    ,       0    1011
 -39         5    9             
  -1  310      48               
  -1   11231      4             
  -1  8                         
 120       -828456      629030      380318     -828456      629030      380318  256       0,   0,       0,WW  ,       0       1
  35       -828456      629030      380318     -828456      689988      380318  256       2,   0,   10000,ELBW,    9000    1011
 -39         3    8             
  -1  086      48               
  -1   11231      4             
  -1  8                         
 150       -828456      689988      380318           0           0           0  256       8,   0,   10000,    ,       0     602
 -22  DOWN                      
 -39         8    2             
  -1  190      48               
  -1   11231      4             
  -1  8                         
 -71  SUPPORT ON CI             
  -1  VIL STANCHION             
 150       -828456      689988      380318           0           0           0  256       9,   0,   10000,01HG,       0     102
 -78  MS007V                    
 -22  NORTH                     
 -39         8                  
  -1  811      48               
  -1   11231      4             
  -1  8                         
  36       -828456      689988      380318     -828456      689988      441277  256       0,   0,   10000,ELBW,    9000    1011
 120       -828456      689988      441277     -828456      689988      441277  256       0,   0,       0,WW  ,       0       1
 100       -828456      689988      441277     -828456      689988      469346  256       1,   0,   10000,    ,       0    1011
 -39         5    8             
  -1  042      48               
  -1   11231      4             
  -1  8                         
 120       -828456      689988      469346     -828456      689988      469346  256       0,   0,       0,WW  ,       0       1
  35       -828456      689988      469346     -828456      689988      530305  256       2,   0,   10000,ELBW,    9000    1011
 -39         3   14             
  -1  266      48               
  -1    9210      4             
  -1  8                         
  36       -828456      689988      530305     -889415      689988      530305  256       0,   0,   10000,ELBW,    9000    1011
 120       -889416      689988      530305     -889416      689988      530305  256       0,   0,       0,WW  ,       0       1
 100       -889416      689988      530305    -1418932      689990      530303  256       1,   0,   10000,    ,       0    1011
 -39         5    8             
  -1  041      48               
  -1    9211      4             
  -1  8                         
 120      -1418932      689990      530303    -1418932      689990      530303  256       0,   0,       0,WW  ,       0       1
  35      -1418932      689990      530303    -1444331      689988      530303  256       7,   0,   10000,ELBW,    4500    1011
 -39         3   14             
  -1  267      48               
  -1    9211      4             
  -1  8                         
  36      -1444331      689988      530303    -1462292      672028      530303  256       0,   0,   10000,ELBW,    4500    1011
 120      -1462293      672027      530303    -1462293      672027      530303  256       0,   0,       0,WW  ,       0       1
 149      -1486594      647726      530304           0           0           0  256       0,   0,       0,FLOW,       0      10
 100      -1462293      672027      530303    -1510895      623425      530306  256       1,   0,   10000,    ,       0    1011
 -39         5    8             
  -1  040      48               
  -1    9211      4             
  -1  8                         
 120      -1510895      623425      530306    -1510895      623425      530306  256       0,   0,       0,WW  ,       0       1
  35      -1510895      623425      530306    -1554000      580320      530306  256       2,   0,   10000,ELBW,    9000    1011
 -39         3   14             
  -1  268      48               
  -1    9212      4             
  -1  8                         
  36      -1554000      580320      530306    -1554000      580320      591265  256       0,   0,   10000,ELBW,    9000    1011
 120      -1554000      580317      591265    -1554000      580317      591265  256       0,   0, 1110000,WW  ,       0       1
 100      -1554000      580317      591265    -1554000      580317     1711261  256       1,   0, 1110000,    ,       0      11
 -30  M/L DWG NO.:              
  -1  AC2Z-P-020-03             
  -1  01-01                     
  47         -4277        4731       80318      -34757        4731       80318  256       1,   0, 1110000,TEBW,       0       1
 -30  M/L DWG NO.:              
  -1  AC21-P-020-03             
  -1  01-02                     
 -20  P01A-0018E                
 -21  A106-B SMLS B             
  -1  36.10M/B36.19             
  -1  M BE S-20                 
 -81  P                         
 -82  S-20                      
 -83  S-20                      
 -84  B1AQ02                    
 -87  P01A-0018E                
 -88  1                         
 -20  P02A-000GM                
 -21  90 DEG ELBOW              
  -1  A234-WPB SMLS             
  -1   B16.9 BW LR              
  -1  S-20                      
 -81  9L                        
 -82  S-20                      
 -83  S-20                      
 -84  B1AQ02                    
 -87  P02A-000GM                
 -88  1                         
 -20  S170X-Z16                 
 -21  GUIDE FOR SHO             
  -1  E                         
 -84  B1AQ02                    
 -87  S170X-Z16                 
 -20  S110X-B16-030             
  -1  0-0100                    
 -21  SHOE                      
 -84  B1AQ02                    
 -87  S110X-B16-030             
  -1  0-0100                    
 -20  S110X-B14-030             
  -1  0-0150                    
 -21  SHOE                      
 -84  B1AQ02                    
 -87  S110X-B14-030             
  -1  0-0150                    
 -20  S131B-B16-010             
  -1  0-FP                      
 -21  DUMMY PIPE                
 -84  B1AQ02                    
 -87  S131B-B16-010             
  -1  0-FP                      
 -20  P03A-000EM                
 -21  45 DEG ELBOW              
  -1  A234-WPB SMLS             
  -1   B16.9 BW LR              
  -1  S-20                      
 -81  4L                        
 -82  S-20                      
 -83  S-20                      
 -84  B1AQ02                    
 -87  P03A-000EM                
 -88  1                         
 -20  S110X-B14-030             
  -1  0-0100                    
 -21  SHOE                      
 -84  B1AQ02                    
 -87  S110X-B14-030             
  -1  0-0100                    
 -20  S131B-B16-030             
  -1  0-FP                      
 -21  DUMMY PIPE                
 -84  B1AQ02                    
 -87  S131B-B16-030             
  -1  0-FP                      
 999
