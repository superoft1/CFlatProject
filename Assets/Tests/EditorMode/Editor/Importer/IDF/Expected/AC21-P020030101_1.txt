Pipe, P01A-0018E, (0.0, 0.0, 1.0), (-11943.0, 9261.0, 104.7), False
(-11943.0, 9261.0, 104.1)
(-11943.0, 9261.0, 105.2)
FreeVertex: 1
Elbow90, P02A-000GM, (-1.0, 0.0, 0.0), (-11943.0, 9261.0, 105.8), False
(-11943.0, 9261.0, 105.2)
(-11942.4, 9261.0, 105.8)
FreeVertex: 0
Pipe, P01A-0018E, (1.0, 0.0, 0.0), (-11941.6, 9261.0, 105.8), False
(-11942.4, 9261.0, 105.8)
(-11940.9, 9261.0, 105.8)
FreeVertex: 0
Pipe, P01A-0018E, (1.0, 0.0, 0.0), (-11938.1, 9261.0, 105.8), False
(-11940.9, 9261.0, 105.8)
(-11935.3, 9261.0, 105.8)
FreeVertex: 0
Elbow90, P02A-000GM, (0.0, -0.7, -0.7), (-11934.7, 9261.0, 105.8), False
(-11935.3, 9261.0, 105.8)
(-11934.7, 9261.5, 106.2)
FreeVertex: 0
Pipe, P01A-0018E, (0.0, 0.7, 0.7), (-11934.7, 9261.7, 106.4), False
(-11934.7, 9261.5, 106.2)
(-11934.7, 9261.9, 106.6)
FreeVertex: 0
Elbow45, P03A-000EM, (0.0, -1.0, 0.0), (-11934.7, 9262.0, 106.8), False
(-11934.7, 9261.9, 106.6)
(-11934.7, 9262.3, 106.8)
FreeVertex: 0
Pipe, P01A-0018E, (0.0, 1.0, 0.0), (-11934.7, 9264.8, 106.8), False
(-11934.7, 9262.3, 106.8)
(-11934.7, 9267.3, 106.8)
FreeVertex: 0
Elbow90, P02A-000GM, (0.0, 0.0, -1.0), (-11934.7, 9267.9, 106.8), False
(-11934.7, 9267.3, 106.8)
(-11934.7, 9267.9, 107.4)
FreeVertex: 0
Pipe, P01A-0018E, (0.0, 0.0, 1.0), (-11934.7, 9267.9, 107.6), False
(-11934.7, 9267.9, 107.4)
(-11934.7, 9267.9, 107.7)
FreeVertex: 0
Elbow90, P02A-000GM, (-1.0, 0.0, 0.0), (-11934.7, 9267.9, 108.3), False
(-11934.7, 9267.9, 107.7)
(-11934.1, 9267.9, 108.3)
FreeVertex: 0
Pipe, P01A-0018E, (1.0, 0.0, 0.0), (-11931.5, 9267.9, 108.3), False
(-11934.1, 9267.9, 108.3)
(-11928.8, 9267.9, 108.3)
FreeVertex: 0
Elbow45, P03A-000EM, (-0.7, 0.7, 0.0), (-11928.6, 9267.9, 108.3), False
(-11928.8, 9267.9, 108.3)
(-11928.4, 9267.7, 108.3)
FreeVertex: 0
Pipe, P01A-0018E, (0.7, -0.7, 0.0), (-11928.1, 9267.5, 108.3), False
(-11928.4, 9267.7, 108.3)
(-11927.9, 9267.2, 108.3)
FreeVertex: 0
Elbow90, P02A-000GM, (0.0, 0.0, -1.0), (-11927.5, 9266.8, 108.3), False
(-11927.9, 9267.2, 108.3)
(-11927.5, 9266.8, 108.9)
FreeVertex: 1
FreeVertexTotal: 2
